### ISSUE TITLE
[Windows][A11y] Windows app doesn't support NVDA/JAWS shortcut `H` for headings

### ISSUE THREAD

**Issue by yimin0613** [flutter/flutter#113960]
See [b/251178462](https://b.corp.google.com/issues/251178462)

## Steps to Reproduce

1. On Windows machine, turn on NVDA/JAWS
2. Execute `flutter run` with the code sample
3. Press shortcut key `H`

**Expected results:** 
3. The heading (Appbar title) should be read out

**Actual results:** 
3. Only "H" is read out instead of the title

<summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';
import 'package:flutter/rendering.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    // Auto-enable accessibility for our Blind and Low Vision customers (see
    // https://docs.flutter.dev/development/accessibility-and-localization/accessibility#screen-readers).
    RendererBinding.instance.setSemanticsEnabled(true);

    return MaterialApp(
      title: 'My Flutter App',
      home: Scaffold(
        appBar: AppBar(
          title: const Text('My Flutter Windows App'),
        ),
        body: Center(
          child: Column(
            children: const [
              Text('This is the first line.'),
              Text('This is the second line.'),
            ],
          ),
        ),
      ),
    );
  }
}
```

**Comment by yaakovschectman** [comment]
Speaking with Nektarios, this may be impossible using MSAA. This functionality may not be possible to add to Flutter Desktop before we migrate its accessibility to use IAccessible2 or, more likely, UI Automation, which would be a larger undertaking.

**Comment by yaakovschectman** [comment]
The listed shortcut here - `H` for headings - is part of NVDA's browse mode. From the NVDA user guide:

```
Complex read-only documents such as web pages are browsed in NVDA using browse mode. This includes documents in the following applications:

- Mozilla Firefox
- Microsoft Internet Explorer
- Mozilla Thunderbird
- HTML messages in Microsoft Outlook
- Google Chrome
- Microsoft Edge
- Adobe Reader
- Foxit Reader
- Supported books in Amazon Kindle for PC

Browse mode is also optionally available for Microsoft Word documents.
```

I suspect this may not be applicable for desktop Flutter apps in general. Is there anyone who would be able to confirm or deny this?

**Comment by gspencergoog** [comment]
Closing because a) it's not clear that this shortcut should work on a non-browser application, and b) it's not feasible to implement it until we are using IAccessible2 or UI Automation, and it's not clear that we will be doing that anytime soon. (Desktop Triage)
